gradle/scripts/tests.gradle

```
@@ -1,28 +1,11 @@
-// Task to run scala tests, as Scala tests are not picked up by gradle by default
-task spec(dependsOn: ['testClasses'], type: JavaExec) {
-	main = 'org.scalatest.tools.Runner'
-	args = [
-		'-R',
-		'build/classes/scala/test',
-		'-h',
-		'build/reports/tests/scalatest',
-		'-oD'
-	]
-	classpath = sourceSets.test.runtimeClasspath
-}
-build.dependsOn spec
+// Currently there are no integration tests
+//task integrationTest(type: Test) {
+//	description 'executes only the integration tests'
+//	filter {
+//		excludeTestsMatching '*Test*.*'
+//		includeTestsMatching '*IT'
+//	}
+//	mustRunAfter test
+//}
 
-task integrationTest(type: Test) {
-	description 'executes only the integration tests'
-	filter {
-		excludeTestsMatching '*Test*.*'
-		includeTestsMatching '*IT'
-	}
-	mustRunAfter test
-}
-
-task allTests(type: Test) {
-	description 'executes all tests'
-	mustRunAfter spec
-}
-allTests.dependsOn(spec)
+check.dependsOn(scalatest)
```

can be removed if `com.github.maiflai.gradle-scalatest.mode = append` is not set 
```suggestion
check.dependsOn(scalatest)
```